modules/ContactsNitroModule/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "contacts-nitro-module",
+  "version": "0.0.1",
+  "main": "src/index",
+  "react-native": "src/index",
+  "description": "React Native Contacts Module with Nitro optimization",
+  "source": "src/index",
+  "files": [
+    "src",
+    "react-native.config.js",
+    "lib",
+    "android/build.gradle",
+    "android/gradle.properties",
+    "android/CMakeLists.txt",
+    "android/src",
+    "ios/**/*.h",
+    "ios/**/*.m",
+    "ios/**/*.mm",
+    "ios/**/*.cpp",
+    "ios/**/*.swift",
+    "app.plugin.js",
+    "*.podspec",
+    "README.md"
+  ],
+  "scripts": {
+    "postinstall": "tsc || exit 0;",
+    "typecheck": "tsc --noEmit",
+    "clean": "del-cli android/build node_modules/**/android/build lib",
+    "lint": "eslint \"**/*.{js,ts,tsx}\" --fix",
+    "lint-ci": "eslint \"**/*.{js,ts,tsx}\" -f @jamesacarr/github-actions",
+    "typescript": "tsc --noEmit false",
+    "specs-debug": "bun run --filter=\"**\" typescript && bun nitro-codegen --logLevel=\"debug\"",
```

Comment:
Let's not introduce bun here. I get that it's fast and developer friendly, but can we just use `npx` instead since that's what we use everywhere else?